unittests/test_data/clair.json

```diff
@@ -0,0 +1,218 @@
+{
+  "clair":{
+    "status": "scanned",
+    "data": {
+      "Layer": {
+        "Name": "sha256:24c0a33d297f04022bf603ad887a7466cdff908caf67794d7acc2e405650e796",
+        "ParentName": "",
```

Comment:
These json data's should be dynamic. These data have to be up-to-date for each new container build. what do you think?

Reply:
They are just some fixture data of policy unittests.

Reply:
I don't think we want to have dynamic data for the static unit tests. We can have a further level of integration tests that would be more in line with the existing solution in #8